lib/features/spaces/widgets/space_card.dart

```diff
@@ -19,6 +19,14 @@ import 'package:totem_app/shared/utils.dart';
 import 'package:totem_app/shared/widgets/space_gradient_mask.dart';
 import 'package:totem_app/shared/widgets/user_avatar.dart';
 
+const _textShadows = [
```

Comment:
`kElevationToShadow` is already constant and should have the same performance impact. I have no problem with this solution, though.